dist/vespa-ann-benchmark.spec

```diff
@@ -15,8 +15,8 @@
 %global __brp_check_rpaths %{nil}
 
 # Force special prefix for Vespa
-%define _prefix /opt/vespa
-%define _command_cmake cmake3
+%define _prefix /opt/vespa-deps
```

Comment:
```suggestion
%define _prefix /opt/vespa
```

Reply:
you need this in %build section before calling cmake:

`export PATH="%{_prefix}-deps/bin:$PATH"`